app/controllers/Checkout.scala

```diff
@@ -57,7 +59,16 @@ object Checkout extends Controller with LazyLogging {
 
     touchpointBackendResolution.backend.checkoutService.processSubscription(formData, idUserOpt).map { case CheckoutResult(_, userIdData, subscription) =>
       val passwordForm = userIdData.toGuestAccountForm
-      Ok(view.thankyou(subscription.name, formData.personalData, passwordForm, touchpointBackendResolution))
+
+      val ratePlan = touchpointBackendResolution.backend.zuoraService.products.filter(_.ratePlanId == formData.ratePlanId).head
+
+      val dataNumberOfMonths = ratePlan.frequency match {
```

Comment:
Feels like this logic should be a method. Something like `ratePlan.frequency.numberOfMonths`